<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LEDControl16GTesterClass</class>
 <widget class="QMainWindow" name="LEDControl16GTesterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LEDControl16GTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButtonReqVersion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req Version</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditVersion">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqAttr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req Attr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDipSW">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>57</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DIPSW</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxTestLED">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>TestLED</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxReset">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>90</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxPLSEnable">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>108</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>PLSEnable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetAttr">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Attr</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPatternNo">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>75</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pattern</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>BaseCounter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>HighCounter</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Target</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Analog</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButtonReqBaseCounter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>710</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req Base counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetBaseCounter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>710</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Base counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqHighCounter">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>750</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req High counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSetHighCounter">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>750</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Set High counter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqAnalog">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>830</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req Analog</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>180</y>
      <width>141</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Comm upper</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditI_IN">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxI_iCLK">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>I_iCLK</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxI_iCmdW">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>I_iCmdW</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxI_iRESET">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>I_iRESET</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxI_iStrobe">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>I_iStrobe</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Comm FPGA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditI_FIO">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>IN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonReqBit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>101</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Req Bit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditKey">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Key</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>141</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Comm upper</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>280</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string>Comm FPGA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxO_InRW">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>O_InRW</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxO_InClk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>O_InClk</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxO_InCS">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>108</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>O_InCS</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditO_FIO">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>320</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>320</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Out</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOutBit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>101</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Out Bit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditO_RPOut">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>71</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Out</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonOutRPOut">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>101</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Out Bit</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSetTargetValue">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>790</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Target</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonReqTargetValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>790</y>
      <width>151</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Req Target</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
